doc/developers/performance.rst

```diff
@@ -344,27 +344,18 @@ Using OpenMP
 Since scikit-learn can be built without OpenMP, it's necessary to protect each
 direct call to OpenMP.
 
-There are some helpers in
+The _openmp_helpers module, available in
 `sklearn/utils/_openmp_helpers.pyx <https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/utils/_openmp_helpers.pyx>`_
-that you can reuse for the main useful functionalities and already have the
-necessary protection to be built without OpenMP.
+provides protected versions of the OpenMP routines. To use OpenMP routines, they
+must be cimported from this module and not from the OpenMP library directly::
```

Comment:
```suggestion
must be C-imported from this module and not from the OpenMP library directly::
```

Reply:
I'm not sure about this one. I find it more intuitive to use the cython keyword.